packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx

```diff
@@ -57,15 +64,29 @@ const VisualizationCardOptions: FC = memo(() => {
                         return {
                             text: 'Area chart',
                             icon: (
-                                <IconChartArea size={20} color={Colors.GRAY1} />
+                                <IconChartArea
+                                    size={20}
+                                    color={
+                                        disabled === true
```

Comment:
same here

Reply:
and basically everywhere you have `disabled === true`

Reply:
ok sure